src/elm/App/Update.elm

```diff
@@ -21,6 +21,10 @@ type Action
   -- NoOp actions
   | NoOp
 
+dragActions : Signal Action
+dragActions =
+    Signal.map ChildTimelineAction Timeline.Update.startTimeActions
```

Comment:
Ah, I guess that's a consequence of what I did here:

https://github.com/Gizra/pow/blob/master/src/elm/Timeline/View.elm#L100-L102

Only an `Element` can be hoverable, so I turned the triangle into an `Element`, but then I had to make that `Element` the same size as the bar in order for the sizing to work right (at least, without further modification). So, I guess it's true that the `hoverable` part is way too big. I'm sure there's a way to fix that -- it would essentially mean doing something more intelligent to make the triangle an Element, so that it's not super-sized in the way I did it. I hadn't actually worked with Graphics.Element before either, so I'm sure there's a better way.